index.js

```diff
@@ -0,0 +1,66 @@
+import displayActive from './modules/navigate.js';
+import BookShelf from './modules/books.js';
+import { DateTime } from './modules/luxon.js';
+
+const booksContainer = document.getElementById('books-dynamic-container');
+const navMenuItem = document.querySelectorAll('.nav-menu-item');
+const addBookForm = document.getElementById('form-add-book');
+const titleInput = document.getElementById('title-input');
+const author = document.getElementById('author-input');
+const dateLocalBox = document.querySelector('.date-local');
+
+const allAddedBooks = new BookShelf();
+
+function insertBooks() {
```

Comment:
- [ ] As it a [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md) for this project, " `Refactor your methods and functions to arrow functions`", please kindly refactor the above function(`insertBooks`) into  an arrow function.